manifold_database_dump_dag.py

```diff
@@ -0,0 +1,94 @@
+"""Airflow DAG to Run the Blogs Sync rake task on a Manifold instance"""
+from datetime import datetime, timedelta
+import airflow
+from airflow.operators.bash_operator import BashOperator
+from airflow.hooks.base_hook import BaseHook
+from airflow.models import Variable
+from airflow.contrib.operators.ssh_operator import SSHOperator
+from airflow.contrib.operators.sftp_operator import SFTPOperator
+from airflow.contrib.operators.sftp_to_s3_operator import SFTPToS3Operator
+# from airflow.operators.python_operator import PythonOperator
+from manifold_airflow_dags.task_slack_posts import slackpostonfail
+# from manifold_airflow_dags.task_slack_posts import slackpostonsuccess
+
+MANIFOLD_INSTANCE_SSH_CONN = \
+        BaseHook.get_connection("AIRFLOW_CONN_MANIFOLD_SSH_INSTANCE")
+MANIFOLD_DUMP_DATABASE_INTERVAL = \
+        Variable.get("MANIFOLD_DUMP_DATABASE_SCHEDULE_INTERVAL")
+DUMP_PATH = "/tmp"
+DUMP_FILENAME = f"manifold-{ datetime.now().strftime('%Y-%m-%dT%H%z')}.sqlc"
```

Comment:
So, each time airflow runs a task in a dag, it loads this dag file and evaluates it. Meaning this `datetime.now()` is going to vary a bit with each task, so the DUMP_FILENAME in the first task may not be the same as the DUMP_FILENAME in the last task. Looks like we are using `execution_date` inside a template elsewere to handle these cases https://github.com/tulibraries/funcake_dags/blob/qa/dev_combine_index_dag.py#L23